vault/external_tests/audit/audit_test.go

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package audit
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/hashicorp/vault/api/auth/userpass"
+	"github.com/hashicorp/vault/helper/testhelpers/minimal"
+	"github.com/stretchr/testify/require"
+)
+
+// TestAudit_HMACFields verifies that all appropriate fields in audit
+// request and response entries are HMACed properly. The fields in question are:
+//   - request.headers.x-correlation-id
+//   - request.data: all sub-fields
+//   - all sub-fields
```

Comment:
Is this line missing something before `all sub-fields`?

Reply:
Oh shoot, let me take a look. Thanks for catching that!